static/src/js/actions/granules.js

```diff
@@ -239,19 +239,26 @@ export const fetchOpendapLinks = retrievalCollectionData => (dispatch, getState)
 
   const { concept_id: excludedGranuleIds = [] } = exclude
 
-  const {
-    swLat,
-    swLng,
-    neLat,
-    neLng
-  } = mbr({
-    boundingBox: boundingBox[0],
-    circle: circle[0],
-    point: point[0],
-    polygon: polygon[0]
-  })
+  if (
+    boundingBox.length
```

Comment:
In the few places that we check these, we use `.length > 0` -- can you make that minor adjustment here? We may eventually want a `hasSpatialFilterApplied` util given how many places were using this, and we have that ticket to add it to the UI as well.